### ISSUE TITLE
Empty VeraCrypt container

### ISSUE THREAD

**Issue by gorkang** [sovworks/edslite#4]
When opening an AES / SHA-512 standard VeraCrypt volume, the app shows the message "The list is empty".

Opening the same volume using VeraCrypt on my computer properly shows the contained files.

**Comment by fractalwrongness** [comment]
Same here. Were you able to resolve it?

**Comment by gorkang** [comment]
Nope. Nothing seems to work, at least for me. Any help would be apreciated.

**Comment by DorKeinath** [comment]
I'm getting an empty list if I changed the folder where to mount to the container.

**Comment by DuckOfDevastation** [comment]
Same problem here: opening a standard Veracrypt container (created with 1.25.9 on Win10, AES, SHA-512) results in "the list is empty". Anyone found a solution in the meantime?

**Comment by sampathkar** [comment]
Same for me :'(
